docs/notebooks/atmodel_tests.ipynb

```diff
@@ -0,0 +1,2097 @@
+{
```

Comment:
I wasn't going to include it at all, it's just easiest for me to keep it there while developing. I have now removed it from the repo.